backend/Match/models.py

```diff
@@ -2,15 +2,18 @@
 from datetime import datetime
 from .matchState import MatchState
 from Tournament.models import Tournament
-class Match(models.Model):
+from django.utils import timezone
+import sys
```

Comment:
Could go

Reply:
shit. Ill take these out on the next PR / create and issue to take out all comments etc.